src/main/asciidoc/jdbc.adoc

```diff
@@ -113,14 +165,24 @@ Everything else is done by Spring Boot.
 
 There are a couple of things one might want to customize in this setup.
 
+[[jdbc.dialects]]
+=== Dialects
+
 Spring Data JDBC uses implementations of the interface `Dialect` to encapsulate behavior that is specific to a database or its JDBC driver.
-By default the `AbstractJdbcConfiguration` tries to determine the database in use an register the correct `Dialect`.
+By default, the `AbstractJdbcConfiguration` tries to determine the database in use an register the correct `Dialect`.
 This behavior can be changed by overwriting `jdbcDialect(NamedParameterJdbcOperations)`.
 
-TIP: Dialects are resolved by [`JdbcDialectResolver`] from `JdbcOperations`, typically by inspecting `Connection`.
+TIP: Dialects are resolved by `JdbcDialectResolver` from `JdbcOperations`, typically by inspecting `Connection`.
 You can let Spring auto-discover your `Dialect` by registering a class that implements `org.springframework.data.jdbc.repository.config.DialectResolver$JdbcDialectProvider` through `META-INF/spring.factories`.
 `DialectResolver` discovers dialect provider implementations from the class path using Spring's `SpringFactoriesLoader`.
 
+If you use a database for which no dialect is available, then your application won’t startup. In that case, you’ll have to ask your vendor to provide a `Dialect` implementation. Alternatively, you can:
```

Comment:
Isn't this the same as the previous paragraph? 
I'd say we keep only one of them.

Reply:
Probably we can remove the previous one as the information is duplicated.